_sass/_donate.scss

```diff
@@ -0,0 +1,21 @@
+body, html {
```

Comment:
Can you move this to `_main.scss` instead so that this style can be shared by all pages? `main.css` is declared in the `base` layout, therefore it will be shared by all pages.

Reply:
I refactored the _donate.scss
The general styling that might be useful for other pages have been moved to _main.scss

I still kept some of the specific ones in _donate.scss though.
How about we discuss the management of scss files on our next meeting too?